### PR TITLE
#1813 Fixed the cards in the Community Spotlight Section

### LINKED ISSUE #1813: [BUG]: Text Overflow in Community Spotlight Cards
### Type

Site Bug Report

### Feedback

**Description**:  
In the Community Spotlight section, the text within the cards is overflowing beyond the card container. This issue impacts the visual layout, causing the cards to appear unaligned and untidy. The text content should remain inside the card container with proper wrapping or truncation to ensure a clean and readable interface.

**Steps to Reproduce**:  
1. Navigate to the Community Spotlight section.
2. Observe the text inside the cards.
3. Notice how longer text strings are overflowing beyond the card borders.

**Expected Behavior**:  
The text should remain within the card container, either by word-wrapping, truncating, or resizing to fit inside the card.

**Actual Behavior**:  
The text overflows outside of the card container, affecting the layout.

### Screenshots

**Screenshot**:
Screenshot (110)

### developer.tbd.website URL

https://developer.tbd.website/community

### Browser

Firefox, Chrome, Microsoft Edge

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by sanjesh17**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [x] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR fixes a bug with the Community Spotlight card where the text was misaligned and did not properly fit within the card container. The text styling has been adjusted to ensure proper alignment and a cleaner design.

## Related Tickets & Documents

Resolves #1813 [BUG]: Text Overflow in Community Spotlight Cards

## Mobile & Desktop Screenshots/Recordings

**Before Fix**

Screenshot (110)

**After Fix**

Screenshot (111)

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### No tests? Add a note

The PR focuses on design adjustments, and tests are not necessary for this change.

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### No docs? Add a note

No documentation needed as this change only affects design.

## [optional] Are there any post-deployment tasks we need to perform?

None.

## [optional] What gif best describes this PR or how it makes you feel?

gif

### CODE REVIEW

File: pnpm-lock.yaml
Comment (angiejones) [inline pnpm-lock.yaml:1]: can you revert changes to this file?

**Review by angiejones — DISMISSED** [review]

File: pnpm-lock.yaml
Comment (sanjesh17) [reply pnpm-lock.yaml:1]: Sure, I'll do it

**Comment by sanjesh17** [comment]
I have reverted the changes, please review it

**Review by EbonyLouis — CHANGES_REQUESTED** [review]
Thank you for working on this, it looks great! However, please adjust the color scheme to what it originally was.

**Comment by sanjesh17** [comment]
> @sanjesh17 Thank you for working on this, it looks great! However, please adjust the color scheme to what it originally was.

Sure, I'll change it accordingly

**Comment by sanjesh17** [comment]
**Made the requested changes**

Screenshot (112)

Please review it..

**Review by EbonyLouis — APPROVED** [review]

**Pull request merged**